### PR TITLE
chore: fusdc `ChainPolicy` schema

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
## Description
- rename `nobleContractAddress` to `attenuatedCttpBridgeAddress` as it better captures the functionality in FUSDC.
  - the contract deployed to this address must be an attenuated wrapper around Circle's TokenMessenger that does not contain `replaceDepositForBurn` . The Oracle Nodes must only report transactions where `depositor` for `DepositForBurn` is this value.
- include jsdoc comments for `ChainPolicy` type

### Security Considerations
None

### Scaling Considerations
None

### Documentation Considerations
Improves docs via jsdoc annotations

### Testing Considerations
None

### Upgrade Considerations
None, unreleased code

### CODE REVIEW

File: packages/fast-usdc/src/types.ts [hunk 1]
```diff
@@ -58,11 +58,14 @@ export interface PoolMetrics extends PoolStats {
 }
 
 export interface ChainPolicy {
-  nobleContractAddress: EvmHash;
+  /** `msg.sender` of DepositAndBurn to TokenMessenger must be an attenuated wrapper contract that does not contain `replaceDepositForBurn` */
```
Comment (turadg) [inline packages/fast-usdc/src/types.ts:61]: 👌

**Review by turadg — APPROVED** [review]

File: packages/fast-usdc/src/type-guards.js [hunk 2]
```diff
@@ -94,16 +94,13 @@ export const PoolMetricsShape = {
...
 
 /** @type {TypedPattern<ChainPolicy>} */
-export const ChainPoliciesShape = M.splitRecord(
-  {
-    nobleContractAddress: EvmHashShape,
-    cctpTokenMessengerAddress: EvmHashShape,
-    confirmations: M.number(),
-    chainId: M.number(),
-  },
-  { chainType: M.number() },
```
Comment (samsiegart) [inline packages/fast-usdc/src/type-guards.js:104]: 👍

File: packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ Generated by [AVA](https://avajs.dev).
         {
           chainPolicies: {
             Arbitrum: {
+              attenuatedCttpBridgeAddress: '0xe298b93ffB5eA1FB628e0C0D55A43aeaC268e347',
```
Comment (samsiegart) [inline packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md:20]: This address is different than the previous `nobleContractAddress`, should we update it in multichain-testing/test/fast-usdc/config.ts as well?

File: packages/fast-usdc/src/utils/deploy-config.js [hunk 4]
```diff
@@ -68,7 +68,8 @@ export const configurations = {
             '0x19330d10D9Cc8751218eaf51E8885D058642E08A',
           chainId: 42161,
           confirmations: 2,
-          nobleContractAddress: '0x19330d10D9Cc8751218eaf51E8885D058642E08A',
+          attenuatedCttpBridgeAddress:
+            '0xe298b93ffB5eA1FB628e0C0D55A43aeaC268e347',
```
Comment (dckc) [inline packages/fast-usdc/src/utils/deploy-config.js]: were these properties previously sorted?

File: packages/fast-usdc/src/types.ts (hunk 1 above)
Comment (dckc) [reply packages/fast-usdc/src/types.ts:61]: I'd like to see a few words from this JSDoc lifed to the PR description and/or title.

not critical.

File: packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md (hunk 3 above)
Comment (0xpatrickdev) [reply packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md:20]: Good catch I am basing this on top of master (instead), but I'll make sure to make this change there too

File: packages/fast-usdc/src/utils/deploy-config.js (hunk 4 above)
Comment (0xpatrickdev) [reply packages/fast-usdc/src/utils/deploy-config.js]: fixed

**Pull request merged**